src/services/currency-converter-uah.js

```diff
@@ -0,0 +1,17 @@
+export class CurrencyConverterUAH {
+  static convert(curr, amount) {
+    let uah;
+    switch (curr) {
```

Comment:
it's an overhead -- just use two `if`s here

Reply:
+